docs/discussions/301.html

```diff
@@ -0,0 +1,167 @@
+                if (window.scrollY === 0) {
+                        backLink.style.display = 'none';
+                } else {
+                        backLink.style.display = 'block';
+                }
+        });
+</script>
+<link rel="stylesheet" type="text/css" href="/style.css">
+<script src="/star.js"></script>
+
+
+
+<h1 id="ex">Getting Started!</h1>
+<a href="/">←Back</a>
+<a href="#" font-style:italic;" onclick="copyToClipboard(window.location.href)">Copy Link</a><br>
+
+<br>
+<h1 id="intro">Intro</h1>
+If you are looking at this, you mayhaps be interesting in getting around network blocks on your device?
+
+If you are simply looking for a Prox(y) setup, this isn't for that;
+<ul>
+    <li>You can either use <a href='https://go.sillylittle.tech'>go.sillylittle.tech</a></li>
+    <li>or</li>
+    <li>append your destination to <a href="https://prox.sillylittle.tech">prox.sillylittle.tech</a> Read about these further below.</li>
+</ul>
+<br>
+<hr class="divider">
+<h1>Proton configs on ChromeOS</h1>
+        <br>
+<h2>Making an IPSec config via Proton</h2>
+<a
+href="https://account.proton.me/vpn/signup?plan=free&billing=12&minimumCycle=12"
+>Make a Proton account</a
+><br />
+A good website to make a burner email for this is
+<a href="https://signup.mail.com/#.7518-header-signup1-1">mail.com</a>(they don't verify info).
+<br>Or you can make your own <a href="https://account.proton.me/mail/signup?plan=free&billing=12&minimumCycle=12">Proton email</a> and use account instead)<br /><br>
+Go to the
+<a href="https://account.proton.me/u/6/vpn/OpenVpnIKEv2"
+>Proton accounts dashboard</a
+>
+<br>
+<img src="https://github.com/dswan36/SillyLittleFiles/assets/75678535/0d5de285-fdb6-4d36-b777-25f1f708f68e" width="600" />
+<br>
+and note your credentials<br />
+<br>Create a file using the <a href='/tools/filecreator.html'>File Creator</a>
+<br><hr class="divider">
+<h1>Importing files into ChromeOS</h1>
+Go to <code>chrome://network</code><br>
+<br>
+Scroll to the bottom and press <code>Import ONC</code>, then upload the onc file<br>
+Delete the file and then go to your quick access settings (click the clock) <br>
+Click <code>VPN</code>, and then the name you selected<br>
+Depending on how used to account is, or how ratedlimited it is. It should a couple seconds. If it dosen't connect right away, try again or wait until later.<br>
+<br>
+<h2>Video on Creating and Importing</h2>
+<video width="500" height="500" controls>
+    <source src="/discussions/google_screen_recording_2024-05-08T04-12_14.958Z.webm" type="video/webm">
+  Your browser does not support the video tag.
+  </video><br><br>
+<hr class="divider">
+
+<h1>Changing your file's region</h1>
+        <br>
+<h2>Getting the hostname</h2>
+        <br>
+<h3>Server Directory</h3>
+You need another hostname or port to change your file's region, these are listed <a href="/tools/directory/serverlister.html">here</a>
+<br>
+        <h3>Manual</h3>
```

Comment:
Add an additional space between manual here
```suggestion
        <br><h3>Manual</h3>
```